src/NuGet.Clients/NuGet.PackageManagement.UI/Options/AddMappingDialog.xaml

```diff
@@ -0,0 +1,133 @@
+<vsui:DialogWindow x:Class="NuGet.Options.AddMappingDialog"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:vsui="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             mc:Ignorable="d" 
+             x:Uid="AddMappingDialogWindow"
+             AutomationProperties.Name="AddMappingDialogWindow"
```

Comment:
Please make sure all accessible names read by Narrator are stored in strings, so they can be localized. 
```suggestion
             AutomationProperties.Name={x:Static nuget:Resources.AddMappingDialogWindow_AccessibleName}
```